From b2df1b836a3cee0293643d4d0f172204a3135fc9 Mon Sep 17 00:00:00 2001
From: Arnaud Pouliquen <arnaud.pouliquen@foss.st.com>
Date: Thu, 2 Jan 2025 18:53:15 +0100
Subject: [PATCH] arm64: dts: st: Remove rx and tx mailbox in m0_rproc in
 stm32mp25x boards

The remoteproc m0 device does not use the rx and tx mailbox channels
for communication with the Cortex-M0.
The shutdown mailbox is already declared in the soc device tree so the
mailbox and mailbox-names properties can be removed.

Change-Id: I2d549b1b94e821a59839d301fe5d46f623bb10dc
Signed-off-by: Arnaud Pouliquen <arnaud.pouliquen@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/426867
Reviewed-by: Gwenael TREUVEUR <gwenael.treuveur@st.com>
Domain-Review: Arnaud POULIQUEN <arnaud.pouliquen@st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Tested-by: Arnaud POULIQUEN <arnaud.pouliquen@st.com>
Reviewed-by: Arnaud POULIQUEN <arnaud.pouliquen@st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
---
 arch/arm64/boot/dts/st/stm32mp257f-dk.dts  | 2 --
 arch/arm64/boot/dts/st/stm32mp257f-ev1.dts | 2 --
 2 files changed, 4 deletions(-)

--- a/arch/arm64/boot/dts/st/stm32mp257f-dk.dts
+++ b/arch/arm64/boot/dts/st/stm32mp257f-dk.dts
@@ -580,8 +580,6 @@
 };
 
 &m0_rproc {
-	mboxes = <&ipcc2 0>, <&ipcc2 1>, <&ipcc2 2>;
-	mbox-names = "rx", "tx", "shutdown";
 	memory-region = <&cm0_cube_fw>, <&cm0_cube_data>;
 	clocks = <&rcc CK_CPU3>,
 		 <&rcc CK_CPU3_AM>,
--- a/arch/arm64/boot/dts/st/stm32mp257f-ev1.dts
+++ b/arch/arm64/boot/dts/st/stm32mp257f-ev1.dts
@@ -622,8 +622,6 @@
 };
 
 &m0_rproc {
-	mboxes = <&ipcc2 0>, <&ipcc2 1>, <&ipcc2 2>;
-	mbox-names = "rx", "tx", "shutdown";
 	memory-region = <&cm0_cube_fw>, <&cm0_cube_data>;
 	clocks = <&rcc CK_CPU3>,
 		 <&rcc CK_CPU3_AM>,
